packages/editor/src/client/atoms/WarningMessage.tsx

```diff
@@ -0,0 +1,24 @@
+import React, {ReactNode} from 'react';
+import { Icon } from 'rsuite';
+import {useTranslation} from 'react-i18next';
+
+
+
+export interface WarningMessageProps {
+  children?: ReactNode
+}
+
+export function WarningMessage({children}: WarningMessageProps) {
```

Comment:
Why didn't you use the `<Message ...>` [component](https://rsuitejs.com/components/message/) from rsuite.js which i recommended in the JIRA [comment](https://wepublish.atlassian.net/browse/WPC-539?focusedCommentId=11473)?